### ISSUE TITLE
Lists does not save correctly

### ISSUE THREAD

**Issue by Leymooo** [timvisee/yaml-wrapper#3]
```java
        YamlConfiguration configuration = new YamlConfiguration();
        configuration.set("list", Arrays.asList("1","2"));
        configuration.set("list-2", Arrays.asList("127.0.0.1", "192.168.88.1"));
        System.out.println(configuration.saveToString());
```
Output will
```
list:
    list:
    - '1'
    - '2'
list-2:
    list-2:
    - 127.0.0.1
    - 192.168.88.1
```

PoC:
![image](https://user-images.githubusercontent.com/8667294/53034845-cbb90480-347c-11e9-8919-5cbaaee1d2d9.png)

**Comment by timvisee** [comment]
Interesting. Thanks a lot for the example, that's definitely a bug.

I'll try to take a look into this later today.

**Comment by timvisee** [comment]
@Leymooo the latest commit 1d7be67ce14492cb047548df7c04458fd07f9a36 includes a fix for this issue.

The fix is versioned `1.2.0` in the maven [repository](https://repo.codemc.org/#browse/browse:maven-public:com%2Ftimvisee%2Fyamlwrapper%2F1.2.0%2Fyamlwrapper-1.2.0.jar). Be sure to update it.

Sorry for the inconvenience.
